### PR TITLE
[OSDOCS#9601]: Release Notes 4.14.12

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-9601

Link to docs preview:
https://71558--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-12

QE review:
N/A

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -3592,3 +3592,46 @@ You can view the container images in this release by running the following comma
...
+$ oc adm release info 4.14.12 --pullspecs
+----
+
+[id="ocp-4-14-12-features"]
+==== Features
+
+The following feature is included in this z-stream release:
+
+[id="ocp-4-14-12-configuring-linuxptp-gm"]
+===== Enabling configuration of Linux PTP services
```
Comment (aireilly) [inline release_notes/ocp-4-14-release-notes.adoc:3618]:
```suggestion
===== ===== Using dual Intel E810 Westport Channel NICs as grandmaster clock with the PTP Operator
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -3592,3 +3592,48 @@ You can view the container images in this release by running the following comma
...
+The host system clock is synchronized from the NIC that is connected to the GNSS time source.
+The second NIC is synced to the 1PPS timing output provided by the NIC that is connected to GNSS.
+For more information see, xref:../networking/ptp/configuring-ptp#configuring-linuxptp-services-as-grandmaster-clock-dual-nic_configuring-ptp[Configuring linuxptp services as a grandmaster clock for dual E810 Westport Channel NICs]. (link:https://issues.redhat.com/browse/TELCODOCS-1480[*TELCODOCS-1480*])
+
+[id="ocp-4-14-12-bug-fixes"]
+==== Bug Fixes
+
+* Previously, the release-to-channel strategy and `oc-mirror` behavior was causing issues for users that perform selective mirroring of our packages. When a user performed selective mirroring of the most recent (and hence default) channel for their package and a new release introduced a new channel, their current default channel became invalid and automatically assigning a new default channel failed. With this release, users can define a `defaultChannel` in the `ImageSetConfig` yaml file, which overrides the `currentlDefault` channel. (link:https://issues.redhat.com/browse/OCPBUGS-28871[*OCPBUGS-28871*])
```
Comment (aireilly) [inline release_notes/ocp-4-14-release-notes.adoc:3628]: Edit for clarity

```suggestion
* Previously, the release-to-channel strategy and `oc-mirror` behavior caused an error with the selective mirroring of packages.
When you selectively mirror the most recent (and hence default) channel for the package and a new release introduces a new channel, the current default channel becomes invalid and the automatic assignment of the new default channel fails.
This problem is now resolved. 
You can now define a `defaultChannel` field in the `ImageSetConfig` CR that overrides the `currentDefault` channel.
(link:https://issues.redhat.com/browse/OCPBUGS-28871[*OCPBUGS-28871*])
```

**Review by aireilly — COMMENTED** [review]
Left a small suggested edit :)

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -3592,3 +3636,6 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.14.0 --pullspecs
 ----
 //replace 4.y.z for the correct values for the release. You do not need to update oc to run this command.
+
+
+
```
Comment (skrthomas) [inline release_notes/ocp-4-14-release-notes.adoc:3641]: Remove whitespace?

**Review by skrthomas — COMMENTED** [review]
/lgtm

File: release_notes/ocp-4-14-release-notes.adoc [hunk 4]
```diff
@@ -3185,6 +3185,55 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+* Previously, the CPU limits from the EFS CSI driver container had the potential to cause performance degradation. With this release, the CPU limits from the EFS CSI driver container have been removed. (link:https://issues.redhat.com/browse/OCPBUGS-28823[*OCPBUGS-28823*])
+
+* Previously, when using the `routingViaHost` mode, access to the `ExternalTrafficPolicy=Local` load balancer services broke. With this release, the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-28789[*OCPBUGS-28789*])
+
+* Previously, when a HostedCluster was deployed and a user defined a KAS `AdvertiseAddress`, it conflicted with the current deployment, overlapping with the other networks like Service, Cluster or Machine network, which caused a deployment failure. With this release, network validations for `AdvertiseAddress` have been added. (link:https://issues.redhat.com/browse/OCPBUGS-20547[*OCPBUGS-20547*])
+
+[id="ocp-4-14-12-updating"]
+==== Updating
+To update an existing {product-title} 4.14 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```
Comment (aireilly) [inline release_notes/ocp-4-14-release-notes.adoc:3235]:
```suggestion
To update an existing {product-title} 4.14 cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```

Comment (obrown1205) [reply release_notes/ocp-4-14-release-notes.adoc]: good catch! tysm!

File: release_notes/ocp-4-14-release-notes.adoc [hunk 5]
```diff
@@ -3185,6 +3185,55 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+* You can now configure `linuxptp` services as grandmaster clock (T-GM) for dual Intel E810 Westport Channel NICs by creating a `PtpConfig` custom resource (CR) that configures both NICs.
+The host system clock is synchronized from the NIC that is connected to the GNSS time source.
+The second NIC is synced to the 1PPS timing output provided by the NIC that is connected to GNSS.
+For more information see, xref:../networking/ptp/configuring-ptp.adoc#configuring-linuxptp-services-as-grandmaster-clock-dual-nic_configuring-ptp[Configuring linuxptp services as a grandmaster clock for dual E810 Westport Channel NICs]. (link:https://issues.redhat.com/browse/TELCODOCS-1480[*TELCODOCS-1480*])
+
+[id="ocp-4-14-12-bug-fixes"]
+==== Bug Fixes
+
+* Previously, the release-to-channel strategy and `oc-mirror` behavior caused an error with the selective mirroring of packages.
+
```
Comment (aireilly) [inline release_notes/ocp-4-14-release-notes.adoc:3221]:
```suggestion
```

**Pull request merged**